src/routes/BuildingsRouter.ts

```diff
@@ -1,18 +1,17 @@
-import { Response, Router } from "express";
+import { Router, Response } from "express";
 import Calculations from "../common/Calculations";
 import Config from "../common/Config";
 import { Globals } from "../common/Globals";
```

Comment:
Could unpack Globals further, that would be nice.

Reply:
Could you explain this one a bit more?

Reply:
Unpack is not the right word, it's called destructuring, and imho it should go deeper into Globals.